packages/cardano-services/src/Handle/TypeOrmHandleProvider.ts

```diff
@@ -0,0 +1,46 @@
+import { Cardano, HandleProvider, HandleResolution, Point, ResolveHandlesArgs } from '@cardano-sdk/core';
+import { HandleEntity } from '@cardano-sdk/projection-typeorm';
+import { In } from 'typeorm';
+import { TypeormProvider, TypeormProviderDependencies } from '../util/TypeormProvider';
+
+export type TypeOrmHandleProviderDependencies = TypeormProviderDependencies;
+
+const handleFields = ['cardanoAddress', 'handle', 'hasDatum', 'policyId'] as const;
+
+type HandleFields = typeof handleFields[number];
+type PartialHandleEntity = Required<Pick<HandleEntity, HandleFields>>;
+
+const handleSelect = <{ [k in HandleFields]: true }>Object.fromEntries(handleFields.map((_) => [_, true]));
```

Comment:
About `PartialHandleEntity`, c7b790c02 should solve this

About `handleSelect`:
- splitting it in multiple lines breaks some SDK eslint rules
- the return type of `fromEntries<true>()` is `{ [k: string]: true; }` it would require a type cast anyway

I searched for some ways to make the return type of `fromEntries()` a bit more strict, but all the solutions I found require some complex type operations... I thought that a simple type cast is a good compromise between readability, simplicity and strictness.

Reply:
yea, it's a good balance then